bin/manala

```diff
@@ -15,8 +15,6 @@ require_once __DIR__.'/../vendor/autoload.php';
 use Manala\Command;
 use Symfony\Component\Console\Application;
 
-define('MANALA_DIR', \Phar::running() ?: __DIR__.'/../');
```

Comment:
Do you just find it convenient? We can live without as there is only one occurrence of `__DIR__` in `src/` for now (should not growth).